src/Symfony/Component/Lock/Store/SemaphoreStore.php

```diff
@@ -15,14 +15,14 @@
 use Symfony\Component\Lock\Exception\InvalidArgumentException;
 use Symfony\Component\Lock\Exception\LockConflictedException;
 use Symfony\Component\Lock\Key;
-use Symfony\Component\Lock\StoreInterface;
+use Symfony\Component\Lock\PersistingStoreInterface;
 
 /**
  * SemaphoreStore is a PersistingStoreInterface implementation using Semaphore as store engine.
  *
  * @author Jérémy Derussé <jeremy@derusse.com>
  */
-class SemaphoreStore implements StoreInterface, BlockingStoreInterface
+class SemaphoreStore implements PersistingStoreInterface, BlockingStoreInterface
```

Comment:
`PersistingStoreInterface` should be removed according to #32803

Reply:
@Simperfit This should still be fixed. :)